### DISCUSSION TITLE
Loss curve issues when training

### DISCUSSION THREAD

**Discussion by ZhihaoXu0313** [deepmodeling/deepmd-kit#1799] [Q&A]
Hi,

I'm running a DP training. When I observed the loss curves from lcurve.out file, I found that my validation loss and training loss are decreasing with training while the RMS error of energy and force never decrease well. It's there any insights about this scenario? Thanks for your help.

Zhihao

**Comment by njzjz** [comment]
The total loss is affected by the prefactor of loss you set.
